fanctrl.py

```diff
@@ -81,14 +132,16 @@ def getMovingAverageTemperature(self, timeInterval):
         for i in range(0, timeInterval):
             tempSum += self.temps[self._tempIndex - i]
         return tempSum / timeInterval
+            
 
     def printState(self):
         print(
-            f"speed: {self.speed}% temp: {self.temps[self._tempIndex]}°C movingAverage: {self.getMovingAverageTemperature(self.movingAverageInterval)}°C"
+            f"speed: {self.speed}% temp: {self.temps[self._tempIndex]}°C movingAverage: {round(self.getMovingAverageTemperature(self.movingAverageInterval), 2)}°C Battery: {self.lastBatteryStatus}"
         )
 
     def run(self, debug=True):
         while True:
+            self.updateStrategy()
```

Comment:
This call should only be made if `strategyOnDischarging` is set, otherwise it's going to do a bunch of things for nothing, including file I/O